src/sqliteformat.cpp

```diff
@@ -1124,11 +1118,9 @@ bool SqliteFormat::Private::insertAttendees(Incidence::Ptr incidence, int rowid)
     const Attendee::List &list = incidence->attendees();
     Attendee::List::ConstIterator it;
     for (it = list.begin(); it != list.end(); ++it) {
-        if (it->email().isEmpty()) {
-            qCWarning(lcMkcal) << "Attendee doesn't have an email address";
+        if (it->email().isEmpty() && it->name().isEmpty()) {
```

Comment:
This is a new restriction. Is it intended, and for what reason ?

Reply:
Should be relaxing the old one. Earlier insertAttendee() got skipped if the item didn't contain email. Now skipping only if it has neither name or email.

Reply:
Oh of course, I should learn to properly read boolean operators...